oeps/architectural-decisions/oep-0058-arch-translations-management.rst

```diff
@@ -189,6 +189,32 @@ reasons that we impact site operators (new translations location), we will have
 development tools as well. In addion, we will create new instructions for developers on
 how to enable translations for a new service/repo when it comes online.
 
+Final State
+***********
+
+Repositories that generate translation files will have their translation files generated
+and pushed to the openedx-translation repository via a github workflow. Once the
+translation files from edx-platform and other repositories are moved to the
+openedx-translations repository, the Transifex GitHub App will link a Transifex project
+of a name such as "Open edX Translations" to the openedx-translations repository. A
+configuration file naming the files that are to be translated and the trigger that pulls
```

Comment:
This configuration file is used by the Transifex GitHub App?  Do you have a link for the app, and the configuration it allows?

Reply:
Here is the [Transifex GitHub App](https://github.com/apps/transifex-integration), and the [configuration](https://docs.transifex.com/transifex-github-integrations/github-tx-ui#linking-a-specific-project-with-a-github-repository) it allows. I'll add them to the OEP too.